mindsdb_native/libs/data_types/transaction_output_row.py

```diff
@@ -36,7 +36,8 @@ def explain(self):
         answers = {}
         for pred_col in self._predict_columns:
             answers[pred_col] = {}
-            prediction_row = {col: self._data[col][self._row_index] for col in self._data.keys()}
+            cols = [col for col in self._data.keys() if 'class_map' not in col]  # filter out class maps
```

Comment:
I'm confused, shouldn't this be `'_class_distribution' not in col` ?

Reply:
Yup, nice catch, this is leftover from a previous commit.